traffic_monitor/cache/astats.go

```diff
@@ -116,8 +117,23 @@ func astatsParse(cacheName string, rdr io.Reader, pollCTX interface{}) (Statisti
 		astats.Ats["system.proc.loadavg"] = astats.System.ProcLoadavg
 		astats.Ats["system.proc.net.dev"] = astats.System.ProcNetDev
 
+		via := ctx.HTTPHeader.Get("Via")
+		if via != "" {
+			result := regexp.MustCompile(` ([a-z0-9\-]*)\..*comcast.net`).FindStringSubmatch(via)
```

Comment:
Also, we can't have `comcast.net` here, this is a shared open source project so there shouldn't be reference to specific users like Comcast.

Reply:
right